x-pack/plugins/xpack_main/public/hacks/telemetry.js

```diff
@@ -76,33 +82,23 @@ export class Telemetry {
           return this._$http(req);
         });
       })
-      .then(response => {
+      // the response object is ignored because we do not check it
+      .then(() => {
         // we sent a report, so we need to record and store the current time stamp
         this._set('lastReport', Date.now());
         this._saveToBrowser();
-        return response;
```

Comment:
I thought this was needed for unit tests - is it not?

Reply:
I haven't updated the tests yet, but we can assert other ways.